build.gradle.kts

```diff
@@ -53,9 +53,6 @@ subprojects {
 
         test {
             useJUnitPlatform()
-            configure<JacocoTaskExtension> {
-                isIncludeNoLocationClasses = true
-            }
```

Comment:
This was an unneeded hold-over from Android `jacoco` configuration, which prevents `jacoco` from working on Java 11+.